src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -158,6 +159,44 @@ public void deleteEventFromPersonList(Event eventToDelete) {
         }
     }
 
+    /**
```

Comment:
Maybe can add in "Updates the event set of person in address book" to make the function of method clearer